### DISCUSSION TITLE
How to open URL？

### DISCUSSION THREAD

**Discussion by cplasfwst** [fyne-io/fyne#4703] [Q&A] [answered]
Can I automatically jump to the browser URL page of my Android device? For example, open https://baicu.com

**Comment by Jacalz** [comment, marked as answer]
Have a look at the `OpenURL()` method on your app variable. It should do the trick: https://pkg.go.dev/fyne.io/fyne/v2#App

**Comment by cplasfwst** [comment]
Thank you
